template/pkg/install.sls

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `topdir` from `tpldir` #}
+{%- set topdir = tpldir.split('/')[0] %}
+{%- from topdir ~ "/map.jinja" import template with context %}
+
+template-pkg:
```

Comment:
More descriptive state name is `template pkg install pkg` per my previous comment.

Reply:
OK.

Reply:
BTW, as this was just refactoring mainly, I didn't want to touch other lines.  But since we've all gone to so much effort here...